packages/ui/src/app/pages/WorkingGroups/WorkingGroupsOpening.tsx

```diff
@@ -127,8 +127,8 @@ export const WorkingGroupOpening = () => {
           <SidePanel neighbor={sideNeighborRef}>
             <ApplicantsList
               allApplicants={opening.applications}
-              myApplication={hiredMember?.member.id === activeMembership?.id ? activeMembership : undefined}
-              hired={hiredMember?.member}
+              myApplication={hiredMember && hiredMember?.member.id === activeMembership?.id ? hiredMember : undefined}
+              hired={hiredMember || undefined}
```

Comment:
Maybe it's OK to allow `undefined | null` in the ApplicantsList component? WDYT? Having to deal with undefined | null is cumbersome and I don't see a big need for this.

We could change the hook above to just `return` so there's no need to write the check.

Reply:
I had to double-check what `hiredMember` is. I think that we could also rename that to something that means "application" not "member".

Reply:
NVM, the logic behind `myMembership` was odd -> it should get your application from all the applications not only from the "hired" one. This was not related to your PR though, so I've changed that.